FlowCrypt/src/test/java/com/flowcrypt/email/security/PgpPwdTest.kt

```diff
@@ -7,26 +7,55 @@ package com.flowcrypt.email.security
 
 import org.junit.Test
 import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import java.lang.IllegalArgumentException
 import java.math.BigInteger
 
 class PgpPwdTest {
     @Test
     fun testEstimateStrength() {
-        val result = PgpPwd.estimateStrength(
+        val actualResult = PgpPwd.estimateStrength(
                 BigInteger("88946283684264"), PgpPwd.PwdType.PASSPHRASE)
-        assertEquals(
-                PgpPwd.PwdStrengthResult(
-                    word = PgpPwd.Word(
-                            match = "week",
-                            word =  "poor",
-                            bar = 30,
-                            color = "darkred",
-                            pass = false
-                    ),
-                    seconds = BigInteger.valueOf(1111829),
-                    time = "2 weeks"
+        val expectedResult = PgpPwd.PwdStrengthResult(
+                word = PgpPwd.Word(
+                        match = "week",
+                        word =  "poor",
+                        bar = 30,
+                        color = "darkred",
+                        pass = false
                 ),
-                result
+                seconds = BigInteger.valueOf(1111829),
+                time = "2 weeks"
         )
+        assertEquals(expectedResult, actualResult)
+    }
+
+    @Test
+    fun testBytesToPassword() {
+        val bytes = byteArrayOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 0, 10, 11, 12, 13, 14, 15)
+        assertEquals("1234-5678-90AB-CDEF", PgpPwd.bytesToPassword(bytes))
```

Comment:
@DenBond7 this is what we use on browser extension when the provided private key is missing a pass phrase - we offer to auto-generate one. Is there a similar mechanism on Android yet? Or do we always ask user to provide a new one?

Reply:
`bytesToPassword()` is not separate funciton in the original source code, but I've moved it here into the separate function to make it easier testable.

Reply:
> @DenBond7 this is what we use on browser extension when the provided private key is missing a pass phrase - we offer to auto-generate one. Is there a similar mechanism on Android yet? Or do we always ask user to provide a new one pass phrase?

We always ask a user to provide a pass phrase

Reply:
Got it. Eventually we may support this scenario to auto-generate for user. Not a priority for now, 2022